api/src/main/java/au/org/aodn/nrmn/restapi/validation/validators/global/formatted/Method3QuadratsSum.java

```diff
@@ -0,0 +1,63 @@
+package au.org.aodn.nrmn.restapi.validation.validators.global.formatted;
+
+import au.org.aodn.nrmn.restapi.model.db.StagedJob;
+import au.org.aodn.nrmn.restapi.model.db.StagedRowError;
+import au.org.aodn.nrmn.restapi.model.db.enums.ValidationLevel;
+import au.org.aodn.nrmn.restapi.validation.StagedRowFormatted;
+import au.org.aodn.nrmn.restapi.validation.validators.base.BaseGlobalFormattedValidator;
+import cyclops.companion.Monoids;
+import cyclops.control.Validated;
+import cyclops.data.tuple.Tuple2;
+import lombok.val;
+
+import java.util.List;
+import java.util.stream.Collectors;
+
+public class Method3QuadratsSum extends BaseGlobalFormattedValidator {
+    public Method3QuadratsSum() {
+        super("method 3");
+    }
+
+    @Override
+    public Validated<StagedRowError, String> valid(StagedJob job, List<StagedRowFormatted> rows) {
+
+        val transectsMap = rows.stream()
+                .filter(row -> row.getMethod().equals(3))
+                .collect(Collectors.groupingBy(row ->
+                        row.getDepth()
+                                + "-"
+                                + row.getSite().getSiteCode()
+                                + "-"
+                                + row.getDate().toEpochDay()
```

Comment:
The depth here doesn't include survey number - so this is actually summing up over all transects not just indivdual ones?

Reply:
This is a formatted row where depth and survey number have been separated!

Reply:
Still missing surveyNum